client/src/pages/Landing.js

```diff
@@ -1,39 +1,38 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
-import { Typography } from "@material-ui/core";
-import { withStyles } from "@material-ui/core/styles";
-import { Route, BrowserRouter } from "react-router-dom";
+import { withStyles } from '@material-ui/core/styles';
+import { Route, BrowserRouter } from 'react-router-dom';
 
-import LogIn from './LogIn.js'
-import SignUp from './SignUp.js'
-import Profile from './Profile.js'
+import Login from './Login.js';
+import SignUp from './SignUp.js';
+import Profile from './Profile.js';
```

Comment:
@mrchrischan @kevncheng @nicboy17 are you all using different linters for the front-end or did you add one more recently? If you are using visual code, you can have one that autocorrects (i.e. prettier.io) after you save

Reply:
added to linter